app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List
+
+
+def calculate_team_total_rating(team: List) -> int:
```

Comment:
Add type annotation to indicate a list of what instances you expect here and in the other methods as well